### PR TITLE
(SIMP-1695) Updated global catalysts

### PR DESCRIPTION
**Pull request by nick-markowski**
SIMP-2156 #close

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -119,14 +119,12 @@
   # CCE-26785-6
   $at_boot = true,
   # CCE-26933-2
-  $to_syslog = defined('$::enable_logging')  ? { true => $::enable_logging,  default => hiera('enable_logging',true) },
+  $syslog = simplib::lookup('simp_options::syslog', { 'default_value' => false, 'value_type' => Boolean } ),
```
Comment (trevor-vaughan) [inline manifests/init.pp]: Just make this strongly typed. You're doing it anyway with the `value_type` and it's both easier and you can ditch the `validate_bool` below if you do that. Not rejecting for this but it'll save a lot of headaches.

File: manifests/config/audisp/syslog.pp [hunk 2]
```diff
@@ -47,18 +47,10 @@
 #     are allowed.
 #
 class auditd::config::audisp::syslog (
-  $drop_audit_logs = true,
-  $priority = 'LOG_INFO',
-  $facility = ''
+  Boolean $drop_audit_logs = true,
+  Enum['LOG_DEBUG', 'LOG_INFO', 'LOG_NOTICE', 'LOG_WARNING', 'LOG_ERR', 'LOG_CRIT', 'LOG_ALERT', 'LOG_EMERG'] $priority = 'LOG_INFO',
+  Enum['', 'LOG_LOCAL0', 'LOG_LOCAL1', 'LOG_LOCAL2', 'LOG_LOCAL3', 'LOG_LOCAL4', 'LOG_LOCAL5', 'LOG_LOCAL6', 'LOG_LOCAL7'] $facility = ''
```
Comment (jeefberkey) [inline manifests/config/audisp/syslog.pp]: Can these be types to keep all these long Enums out of the manifest?

File: manifests/config/audisp.pp [hunk 3]
```diff
@@ -13,18 +13,13 @@
 # Trevor Vaughan <tvaugan@onyxpoint.com>
 #
 class auditd::config::audisp (
-  $q_depth = '160',
-  $overflow_action = 'syslog',
-  $priority_boost = '4',
-  $max_restarts = '10',
-  $name_format = 'user',
-  $specific_name = $::fqdn
+  Stdlib::Compat::Integer $q_depth = '160',
```
Comment (jeefberkey) [inline manifests/config/audisp.pp]: Vertically align $ signs

File: manifests/config/audisp/syslog.pp (hunk 2 above)
Comment (nick-markowski) [inline manifests/config/audisp/syslog.pp]: So I have to make a new type every time something is long?

**Pull request merged**